website/pages/ar/deploying/deploy-using-subgraph-studio.mdx

```diff
@@ -0,0 +1,139 @@
+- Create and manage your API keys for specific subgraphs
+- Restrict your API keys to specific domains and allow only certain Indexers to query with them
+- Create your subgraph through the Studio UI
+- Deploy your subgraph using the The Graph CLI
+- Test your subgraph in the playground environment
+- Integrate your subgraph in staging using the development query URL
+- Publish your subgraph with the Studio UI
+- Manage your billing
+
+## Install The Graph CLI
+
+Before deploying, you must install The Graph CLI.
+
+You must have [Node.js](https://nodejs.org/) and a package manager of your choice (`npm`, `yarn` or `pnpm`) installed to use The Graph CLI. Check for the [most recent](https://github.com/graphprotocol/graph-tooling/releases?q=%40graphprotocol%2Fgraph-cli&expanded=true) CLI version.
+
+**Install with yarn:**
+
+```bash
+yarn global add @graphprotocol/graph-cli
+```
+
+**Install with npm:**
+
+```bash
+npm install -g @graphprotocol/graph-cli
+```
+
+## Create Your Subgraph
+
+Before deploying your subgraph you need to create an account in [Subgraph Studio](https://thegraph.com/studio/).
+
+1. Open [Subgraph Studio](https://thegraph.com/studio/).
+2. Connect your wallet to sign in.
+   - You can do this via MetaMask, Coinbase Wallet, WalletConnect, or Safe.
+3. After you sign in, your unique deploy key will be displayed on your subgraph details page.
+   - The deploy key allows you to publish your subgraphs or manage your API keys and billing. It is unique but can be regenerated if you think it has been compromised.
+
+> Important: You need an API key to query subgraphs
+
+### How to Create a Subgraph in Subgraph Studio
+
+<VideoEmbed youtube="nGIFuC69bSA" />
+
+> For additional written detail, review the [Quick-Start](/quick-start/).
+
+### Subgraph Compatibility with The Graph Network
+
+In order to be supported by Indexers on The Graph Network, subgraphs must:
+
+- Index a [supported network](/developing/supported-networks)
+- Must not use any of the following features:
+  - ipfs.cat & ipfs.map
+  - Non-fatal errors
+  - Grafting
+
+## Initialize Your Subgraph
+
+Once your subgraph has been created in Subgraph Studio, you can initialize its code through the CLI using this command:
+
+```bash
+graph init --studio <SUBGRAPH_SLUG>
+```
+
+You can find the `<SUBGRAPH_SLUG>` value on your subgraph details page in Subgraph Studio, see image below:
+
+![Subgraph Studio - Slug](/img/doc-subgraph-slug.png)
+
+After running `graph init`, you will be asked to input the contract address, network, and an ABI that you want to query. This will generate a new folder on your local machine with some basic code to start working on your subgraph. You can then finalize your subgraph to make sure it works as expected.
+
+## Graph Auth
+
+Before you can deploy your subgraph to Subgraph Studio, you need to login into your account within the CLI. To do this, you will need your deploy key, which you can find under your subgraph details page.
```

Comment:
```suggestion
Before you can deploy your subgraph to Subgraph Studio, you need to log into your account within the CLI. To do this, you will need your deploy key, which you can find under your subgraph details page.
```

Reply:
Ugh, just realized I left these comments in the Arabic version of the page... I really need to fix the build so we don't need to duplicate pages, it makes PRs so messy haha. Sorry about that!